test/features/dictionary_objects/values.js

```diff
@@ -1,4 +1,4 @@
-let e = {};
+const e = {};
```

Comment:
Maybe it is still better to use `let` here, because this object is being changed in the next line (adding property)?